### ISSUE TITLE
Starting a second Jupynium instance

### ISSUE THREAD

**Issue by MihailBogojeski** [kiyoon/jupynium.nvim#113]
As far as I can tell, whenever jupynium is called it checks if there is a jupynium instance running, and prints "Jupynium is already running. Attach nvim using --nvim_listen_addr". It would be nice to have an option to start another jupynium instance on a different notebook URL. This would enable us to have to notebook servers running in parallel, which would be useful when working on two different projects at the same time.

Ideally, if the command jupynium --notebook_URL [...] is called with a URL different than any currently running jupynium instance, a new instance will be started on this notebook URL. Then, when calling :JupyniumAttachtToServer, we should also be able to select which instance we want to attach to.

Maybe I am wrong and this is already possible, but as far as I could tell this was not the case.

**Comment by kiyoon** [comment]
Hi, if you mean you want to connect to another Jupyter Notebook (instead of starting another Jupynium which will start another Firefox) in a new tab, I think you can just call `jupynium --notebook_URL localhost:8889` and it will just open that URL no problem.
